scripts/find_headers.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
```

Comment:
This file needs the license header, including your copyright, as indicated in the committer agreement.